doc/rados/operations/stretch-mode.rst

```diff
@@ -9,7 +9,7 @@ Stretch Clusters
 ================
 Ceph generally expects all parts of its network and overall cluster to be
 equally reliable, with failures randomly distributed across the CRUSH map.
-So you may lose a switch that knocks out a big segment of OSDs, but we expect
+So you may lose a switch that knocks out a swath of OSDs, but we expect
```

Comment:
s/swath/number/

I just think the mowing metaphor here isn't as direct as the less-evocative, "number".

Reply:
ok